tests/unit/v1/test_transaction.py

```diff
@@ -992,25 +1045,23 @@ def _make_client(project="feral-tom-cat"):
 
 
 def _make_transaction(txn_id, **txn_kwargs):
-    from google.protobuf import empty_pb2
-    from google.cloud.firestore_v1.gapic import firestore_client
-    from google.cloud.firestore_v1.proto import firestore_pb2
-    from google.cloud.firestore_v1.proto import write_pb2
+    from google.protobuf import empty_pb2 as empty_pb2
```

Comment:
Is there a difference between this and `from google.protobuf import empty_pb2`?

Reply:
This is likely due to the way the fixup scripts work. I can simplify this.